docs/migration-guides/1.19-1.20.md

```diff
@@ -0,0 +1,82 @@
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: logging
+    kyma-project.io/installation: ""
+data:
+  fluent-bit.conf.Output.http.enabled: "true"
+  fluent-bit.conf.Output.http.Match : "*"
+  fluent-bit.backend.http.host: "<Fluentd-endpoint from service key json>"
+  fluent-bit.backend.http.port: "443"
+  fluent-bit.backend.http.http_user: "<Fluentd-username from service key json>"
+  fluent-bit.backend.http.http_passwd: "<Fluentd-password from service key json>"
+  fluent-bit.backend.http.tls: "true"
+  fluent-bit.backend.http.tls_verify : "false"
+```
+
+Old overrides
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: logging
+    kyma-project.io/installation: ""
+data:
+  fluent-bit.config.outputs.http.enabled: "true"
+  fluent-bit.config.outputs.http.match : "*"
+  fluent-bit.config.outputs.http.host: "<Fluentd-endpoint from service key json>"
+  fluent-bit.config.outputs.http.port: "443"
+  fluent-bit.config.outputs.http.httpUser: "<Fluentd-username from service key json>"
+  fluent-bit.config.outputs.http.httpPasswd: "<Fluentd-password from service key json>"
+  fluent-bit.config.outputs.http.tls.enabled: "true"
+  fluent-bit.config.outputs.http.tls.verify : "false"
+```
+
+New overrides
+
+This configuration can also be passed using the new additional keyword. Additional keyword exists also for inputs, filters, and parsers.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: logging
+    kyma-project.io/installation: ""
+data:
+  fluent-bit.config.outputs.additional: |
+      [OUTPUT]
+          Name                        http
+          Match                       *
+          Host                        <Fluentd-endpoint from service key json>
+          Port                        443
+          HTTP_User                   <Fluentd-username from service key json>
+          HTTP_Passwd                 <Fluentd-password from service key json>
+          tls                         On
+          tls.verify                  false
+```
+
+Additionally, we provide a way to keep the sensitive information - such as credentials for an output plugin - to be deployed inside a Secret instead of ending up in a ConfigMap on the cluster. You can pass the secret values as key/value pairs [here](https://github.com/kyma-project/kyma/blob/master/resources/logging/charts/fluent-bit/values.yaml#L135) and they will be populated to the Fluent Bit Pods as environment variables mounted from a Secret. Then, you can use the environment variable in the Fluent Bit configuration. We strongly recommend moving all the sensitive information in your configuration to this new field as they are kept more secure in a Secret.
```

Comment:
"Information" is singular, so the verb should be in singular as well (unless this refers to the environment valuables?)
```suggestion
Additionally, we provide a way to keep the sensitive information - such as credentials for an output plugin - to be deployed inside a Secret instead of ending up in a ConfigMap on the cluster. You can pass the secret values as key/value pairs [here](https://github.com/kyma-project/kyma/blob/master/resources/logging/charts/fluent-bit/values.yaml#L135) and they will be populated to the Fluent Bit Pods as environment variables mounted from a Secret. Then, you can use the environment variable in the Fluent Bit configuration. We strongly recommend moving all the sensitive information in your configuration to this new field as it is kept more secure in a Secret.
```